source/Halibut.Tests/Support/ProxyFactory.cs

```diff
@@ -0,0 +1,24 @@
+using Halibut.TestProxy;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Serilog.Extensions.Logging;
+
+namespace Halibut.Tests.Support
+{
+    public class ProxyFactory
+    {
+        HttpProxyOptions options = new HttpProxyOptions();
+        SerilogLoggerFactory serilogLoggerFactory = new SerilogLoggerFactory(new SerilogLoggerBuilder().Build());
+        bool delaySendingSectionsOfHttpHeaders = true;
```

Comment:
What am I missing here!! `delaySendingSectionsOfHttpHeaders` isn't used!!

Reply:
:facepalm: ugh failed to refactored!

Reply:
That does work now:
![image](https://github.com/OctopusDeploy/Halibut/assets/7076477/b4bcb31c-c2c8-4d3f-8797-5aa0365a928e)

You can see if the proxy builder is told to always use this then tests do fail